eventsource/eventsource-prototype.any.js

```diff
@@ -0,0 +1,10 @@
+// META: title=EventSource: prototype et al
+
+      test(function() {
+        EventSource.prototype.ReturnTrue = function() { return true }
+        var source = new EventSource("resources/message.py")
+        assert_true(source.ReturnTrue())
+        assert_own_property(window, "EventSource")
```

Comment:
```suggestion
        assert_own_property(self, "EventSource")
```
If you want this to run in workers (or rename the file).